### PR TITLE
MF-640 - Clean up the Contribution guide

### LINKED ISSUE #640: Clean up the Contribution guide
**BUG REPORT**

Contribution guide has a few grammar mistakes and certain sentences could be rephrased to make it sound better.

### PR DESCRIPTION
**Pull request by alwindoss**
Signed-off-by: Alwin Doss <alwindoss84@gmail.com>

### What does this do?

- Corrected a few grammar mistakes
- Rephrased a few sentences to read better

### Which issue(s) does this PR fix/relate to?
Put here `Resolves #640` to auto-close the issue that your PR fixes (if such)

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
Not Applicable

### Did you document any new/modified functionality?
No

### Notes
NA

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,7 +46,7 @@ cd mainflux
 git remote add upstream https://github.com/mainflux/mainflux.git
 ```
 
-If you cloned a while ago, get the latest changes from upstream:
+If you cloned repository is behind the upstream commits than get the latest changes from upstream:
```
Comment (anovakovic01) [inline CONTRIBUTING.md]: `If your cloned repository is behind the upstream commits, then get the latest changes from upstream:`

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by chombium — DISMISSED** [review]
Thanks for cleaning up the doc @alwindoss

File: CONTRIBUTING.md (hunk 1 above)
Comment (alwindoss) [reply CONTRIBUTING.md]: Took care of this. Thanks @anovakovic01

**Review by anovakovic01 — APPROVED** [review]
@alwindoss Thank you for your contribution! LGTM!

**Review by manuio — APPROVED** [review]
LGTM

**Comment by drasko** [comment]
@alwindoss thank a lot for this contribution - English is not our mother thnogue, so this is really imporant.

In any case, welcome to Mainflux contributors club ;).

**Comment by alwindoss** [comment]
@drasko and everyone,

I am a Go programmer, but I would really love to review the documentation too and raise PRs as and when I make them. Please do add me to any group you might have so we can discuss this more. I am very very happy to help and even more excited about contributing.

**Comment by drasko** [comment]
@alwindoss best way to reach us is via project's official chat: https://gitter.im/Mainflux/mainflux

There is also Google group as a mailing list, but it is not so frquent: https://groups.google.com/forum/#!forum/mainflux

All contributions and help is very welcome!

**Pull request merged**